src/platform-apps/channels/channel-list.tsx

```diff
@@ -28,18 +28,30 @@ export class ChannelList extends React.Component<Properties> {
     return (
       <div className='channel-list__category-channel-list'>
         {channels.map((channel) => {
-          const className = classNames('channel-list__channel', {
+          const { unreadCount } = channel;
+
+          let unreadCountLabel = unreadCount + '';
+          if (unreadCount > 9) {
+            unreadCountLabel = '9+';
+          }
+
+          const className = classNames('channel-list__channel-info', {
```

Comment:
let's add some tests for the count display. at least tests for no unread messages, a number between 1 and 9, and more than 9.